### ISSUE TITLE
Add implementation for VertexAI (Google/Gemini) representation model

### ISSUE THREAD

**Issue by aitorgp** [MaartenGr/BERTopic#2246]
### Feature request

There are representation model implementations for several providers and flavors (OpenAI, Cohere, llamacpp, ...), but I haven't found a way to use vertexai to invoke Gemini from Google. It would basically the same as the OpenAI implementation, but with another LLM-provider client and a few different details.

### Motivation

I tested the representation of the topics using OpenAI client, and the results are very nice.
In my current company I have access to Gemini through a vertexai client, and I did not found a way to use it in the same way than OpenAI (and other providers) can be used for the representation of the topics.

### Your contribution

I have implemented this missing BaseRepresentation class for vertexai, imitating the one already available for OpenAI, just changing the required parts.
I have tested it in the project I am working at, invoking gemini-1.5-flash-002 model, and it works just like ChatGPT does with the OpenAI client.

```python
vertexai.init(xxxxx)  # whatever let's you to authenticate for the use of the VertexAI API
representation_model = VertexAIRepresentation(model_name="gemini-1.5-flash-002")  # all the init parameters have sensible defaults
topic_model = BERTopic(... [other params here] ..., representation_model=representation_model)
````

I could contribute the implemented class (it extends the BaseRepresentation class just like the others), in case it is useful for somebody else (it is a standalone class, so it will not disrupt anything).

**Comment by MaartenGr** [comment]
Thanks for the extensive description and clear request!

Before going into this specific model, would it also suffice if you were to use `LiteLLM` for this? I [recently](https://github.com/MaartenGr/BERTopic/pull/2213) added it as a representation model that allows you to select VertexAI (among others).

**Comment by aitorgp** [comment]
Hi Maarten,

Oh, yes, the LiteLLM implementation does work for vertex too! I totally overlooked it when I was browsing the code and the documentation, my bad.
(It is a bit hidden, probably I saw the example using OpenAI and got mislead) :-)

I have just tested the exact same code and no problem at all, same result, with good-looking topic labels.

For reference, in case some other person comes after the same, I just needed this to invoke de LiteLLM version:

```python
from bertopic.representation import LiteLLM

project_id = "XXXXXX"
region = "XXXXXX"
vertexai.init(project=project_id, location=region)
representation_model = LiteLLM(model="gemini-1.5-flash-002")
```

Thank you for the information! No need to add any extra class then. I close the issue.

P.S. Congrats for all the great work :-)
